### ISSUE TITLE
Wrong world map country borderline

### ISSUE THREAD

**Issue by UnforbiddenYet** [zcreativelabs/react-simple-maps#237]
First-off, thanks for the great package!

The issue with world map borderline rendering was discovered when trying out the demos on the landing page.

The Ukrainian territory of Crimean Peninsula is displayed as not belonging to the country, which by all means is incorrect and must be changed to comply with the Ukrainian and international laws.

**Comment by skolodyazhnyy** [comment]
Also, the map source you are referencing "Natural Earth data" does not exist anymore. Maybe you should find a different source for the map data.

**Comment by TheQuietOwl** [comment]
Yeah, I'm getting complaints about this as well. Any possibility we can change the json file to add the Crimean peninsula to Ukraine instead of Russia?

**Comment by polianych** [comment]
Confirm. I use topojson from this repo https://github.com/deldersveld/topojson

**Comment by nfx** [comment]
address this issue ASAP.
